stcgal/protocols.py

```diff
@@ -1653,6 +1693,7 @@ def program_options(self):
 
         print("Target UID: %s" % Utils.hexstr(self.uid))
 
+
```

Comment:
I added a new empty line because there are always two lines between two class definitions except for this one.  it is indeed irrelevant to the functionality of this commit.